documentation/blog/2025-12-19-goose-mobile-terminal/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Goose mobile access and native terminal support"
+description: "2 new ways to use goose"
+authors: 
+    - mic
+---
+
+![Goose on iOS - access your personal desktop agent from anywhere](mobile_shots.png)
+
+# 2 new ways to use goose
+
+We're excited to announce two new ways to interact with goose: a native iOS app for mobile access and native terminal integration. Both give you more flexibility in how and where you use your AI agent.
+
+<!-- truncate -->
+
+## Goose iOS App
+
+Goose AI is now available on the App Store! The iOS app connects to your desktop goose instance via a secure tunnel, letting you interact with your agent from anywhere.
+
+<a href="https://apps.apple.com/au/app/goose-ai/id6752889295">
+  <img src="https://tools.applemediaservices.com/api/badges/download-on-the-app-store/black/en-us?size=250x83" alt="Download on the App Store" />
+</a>
+
+### Getting Started with Mobile
+
+1. **Install the app** - Download [Goose AI from the App Store](https://apps.apple.com/au/app/goose-ai/id6752889295)
+2. **Enable remote access** - In the Goose desktop app, go to App Settings and turn on "Remote Access"
+3. **Scan the QR code** - Use the iOS app to scan the QR code displayed in your desktop app
+4. **Start working** - You're connected! Your mobile app now tunnels to your desktop goose instance
+
+This means you get the full power of your desktop goose setup—all your extensions and configurations—accessible from your phone. Whether you're on the train, grabbing coffee, or just away from your desk, you can still ask goose to help with tasks or check on long running things. Throw an idea out there for it to go to work on and pick it up later.
+
+The Goose iOS app also runs natively on macOS (Apple Silicon Macs), giving you another lightweight option for accessing your goose instance from another device.
+
+## Native Terminal Support
+
+At the other end of things, there is a brand new way to use goose natively in your favoured terminal. 
```

Comment:
we have documentation on this..lets link to the docs here.

Reply:
done.